columnflow/production/cms/btag.py

```diff
@@ -210,7 +249,3 @@ def btag_weights_setup(
     )
     corrector_name = self.get_btag_config()[0]
     self.btag_sf_corrector = correction_set[corrector_name]
-
-    # check versions
-    if self.btag_sf_corrector.version not in (3,):
-        raise Exception(f"unsuppprted btag sf corrector version {self.btag_sf_corrector.version}")
```

Comment:
Why did we need this in the first place? Might this break something again when we remove this?